playwright-e2e/tests/create-case.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from "@playwright/test";
```

Comment:
Hi @Braimah101 is there a reason this test has been added back in? we did remove it, it is also currently skipped.